src/constatnts.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable max-len */
+const STATIC_CASES = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
```

Comment:
Maybe it is better to rename this variable `POSSIBLE_STYLES` , now it is not clear which cases we are talking about